Dockerfile.desktop

```diff
@@ -0,0 +1,97 @@
+FROM ubuntu:lunar
+
+USER root
+
+# Set env
+ENV DEBIAN_FRONTEND=noninteractive \
+    MOZ_HEADLESS=1 \
+    LC_ALL=C.UTF-8 \
+    LANG=C.UTF-8 \
+    FIREFOX_VERSION=112.0 \
+    GECKODRIVER_VERSION=0.33.0
+
+# Install requirements to install tools
+RUN dependencies=' \
+        bzip2 \
+        ca-certificates \
+        curl \
+        firefox \
+        python3-pip \
+        python3-setuptools \
+        python3-wheel \
+        sudo \
+        wget \
+        build-essential \
+        gcc \
+        python3 \
+        tox \
+        pipx \
+        libgtk-3-dev \
+        libasound2 \
+        libdbus-glib-1-dev \
+    ' \
+    && set -x \
+    && apt-get -qq update && apt-get -qq install -y software-properties-common \
+    && apt-get -qq update && apt-get -qq install --no-install-recommends -y $dependencies \
+    && apt-get -y purge firefox \
+    && rm -rf /var/lib/apt/lists/* /var/cache/apt/*
+
+ENV VIRTUAL_ENV=/opt/venv
+RUN python3 -m venv $VIRTUAL_ENV
+ENV PATH="$VIRTUAL_ENV/bin:$PATH"
+
+# Install Firefox and Geckodriver
+RUN INSTALLER_DOWNLOAD_URL=https://raw.githubusercontent.com/hackebrot/install-firefox/master/install-firefox.sh \
+    && curl $INSTALLER_DOWNLOAD_URL -sSf | sh -s -- --firefox $FIREFOX_VERSION --geckodriver $GECKODRIVER_VERSION
```

Comment:
So the `FIREFOX_VERSION` and `GECKODRIVER_VERSION` from this file, override the contents of the GitHub repo above on L44?
> ```ini
> FIREFOX_VERSION="59.0"
> GECKODRIVER_VERSION="0.20.0"
> ```